### ISSUE TITLE
Include a Thousand Validator identifier on Validator's profile page.

### ISSUE THREAD

**Issue by paradox-tt** [Colm3na/polkastats-ng#1]
Dear Polkastat.io,

I would like to suggest that on the respective [validator profile page](https://polkadot.polkastats.io/validator/14hM4oLJCK6wtS7gNfwTDhthRjy5QJ1t3NAcoPjEepo9AH67) that you include an identifier indicating that the validator is a member of the Thousand Validator Programme (TVP).

It is my opinion that this additional bit of information would be useful to nominators as members of the TVP are upheld to performance standards such as controls on their commission, no slash history, certainty of pay-outs and enter the active set periodically.

The W3F provides endpoints which list the members of the programme on Kusama and on Polkadot found https://kusama.w3f.community/candidates and here https://polkadot.w3f.community/candidates respectfully.

Your feedback would be most appreciated.

Regards,
Paradox

**Comment by michaelmowry** [comment]
I agree and think this would be good information for viewers. Perhaps the indicator could also hyperlink to more information on what it means to be a 1KV validator.
